libs/core/langchain_core/runnables/fallbacks.py

```diff
@@ -343,3 +347,74 @@ async def abatch(
             raise ValueError("No error stored at end of fallbacks.")
         await asyncio.gather(*(rm.on_chain_error(first_error) for rm in run_managers))
         raise first_error
+
+    def stream(
+        self,
+        input: Input,
+        config: Optional[RunnableConfig] = None,
+        **kwargs: Optional[Any],
+    ) -> Iterator[Output]:
+        # setup callbacks
```

Comment:
Would you mind adding a doc-string here to document the semantics a bit better; i.e., what happens if the exception it not during the first chunk?